vm/gce/gce.go

```diff
@@ -108,6 +108,10 @@ func Ctor(env *vmimpl.Env, consoleReadCmd string) (*Pool, error) {
 	}
 
 	GCE, err := gce.NewContext(cfg.ZoneID)
+	if err != nil {
```

Comment:
Is there any downside of retrying more?
I can only imagine slightly increased delay before producing an error, when syz-manager mis-configured and only during initial bring up.
If this happens with considerable probability, it can sense to retry more.

Reply:
With high probability gce.NewContext() generates some requests to GCE Load Balancer.
I expect LB to stop the faulty backend usage after the first error.
Single retry will help a lot in such situation. But it is an assumptions based opinion.

Reply:
I think doing it too many times won't help either -- if GCE API is already temporarily broken for some reason, we will only make it worse for them by querying them from all our syz-managers.

Reply:
I've changed the code to retry 3 times and added backoffs.

PTAL